src/BaseTable.tsx

```diff
@@ -428,6 +421,7 @@ class BaseTable extends React.PureComponent<IBaseTableProps, IBaseTableState> {
         rowRenderer={this.renderRow}
         onScroll={this._handleScroll}
         onRowsRendered={this._handleRowsRendered}
+        columnWidth={DEFAULT_COLUMN_WIDTH}
```

Comment:
In interface `FixedSizeGridProps`, of react-windows type defs, `columnWidth` is a required property. This interface is being inherited `GridTableProps`. 

(In fact, I am removing columnWidth from GridTableProps since it is already there in FixedSizeGridProps.